R/database.R

```diff
@@ -112,36 +126,40 @@ get_ssm_by_patients = function(these_patient_ids,
                             engine=engine))
 }
 
-
-#' Get MAF-format data frame for more than one sample and combine together
+#' @title Get SSM By Samples.
+#'
+#' @description Get MAF-format data frame for more than one sample and combine them together.
 #'
-#' This function internally runs get_ssm_by_sample.
-#' In most situations, this should never need to be run with subset_from_merge = TRUE. Instead use one of get_coding_ssm or get_ssm_by_region
-#' See get_ssm_by_sample for more information
+#' @details This function internally runs `get_ssm_by_sample`.
+#' In most situations, this should never need to be run with subset_from_merge = TRUE. 
+#' Instead use one of `get_coding_ssm` or `get_ssm_by_region`.
+#' See `get_ssm_by_sample` for more information.
 #'
 #' @param these_sample_ids A vector of sample_id that you want results for.
-#' @param these_samples_metadata Optional metadata table
-#' @param tool_name Only supports slms-3 currently
-#' @param augmented default: TRUE. Set to FALSE if you instead want the original MAF from each sample for multi-sample patients instead
-#' @param projection Obtain variants projected to this reference (one of grch37 or hg38)
-#' @param flavour Currently this function only supports one flavour option but this feature is meant for eventual compatability with additional variant calling parameters/versions
+#' @param these_samples_metadata Optional metadata table.
```

Comment:
As in my last comment, please confirm the function works with either these_sample_ids or these_samples_metadata such that either of them can be provided to ensure the function runs only on the IDs the user wants. As long as this is how it works, please update the documentation so this is clear.

Reply:
I can confirm it works as advertised. I have also updated the documentation to make this clear as well as added an additional example demonstrating how to use this function with an already subset metadata table.